composer.json

```diff
@@ -10,13 +10,14 @@
     ],
     "prefer-stable": true,
     "require": {
-        "php": ">= 8.0.2",
+        "php": ">= 8.2",
         "algolia/algoliasearch-client-php": "^3.0",
-        "doctrine/event-manager": "^1.1",
+        "doctrine/annotations": "^2.0",
```

Comment:
is doctrine/annotations really needed here? i see that you replaced annotations with attributes.

Reply:
You're right, it's not! Thanks for spotting!